cmake/CustomInstallDirs.cmake

```diff
@@ -13,6 +13,9 @@ set(RATS_INSTALL_LIBV_PATH "${RATS_INSTALL_LIB_PATH}/verifiers")
 # include/rats
 set(RATS_INSTALL_INCLUDE_PATH "${RATS_INSTALL_PATH}/include/")
 
+# /usr/share/rats
+set(RATS_INSTALL_BIN_PATH "/usr/share/librats/samples")
```

Comment:
For users, we will provide deb package rpm package for them to use. For our developers, I think it will be more convenient to install samples into the system.